types/nodemailer-mailgun-transport/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for nodemailer-mailgun-transport 1.3
+// Project: https://github.com/orliesaurus/nodemailer-mailgun-transport
+// Definitions by: Oto Ciulis <https://github.com/otociulis/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as nodemailer from 'nodemailer';
+
+declare namespace mailgunTransport {
+    interface AuthOptions {
+        api_key: string;
+        domain?: string;
+    }
+}
+
+declare function mailgunTransport(options: mailgunTransport.AuthOptions): nodemailer.Transport;
```

Comment:
Is the return type correct as nodemailer.Transport? It seems like in your implementation(https://github.com/orliesaurus/nodemailer-mailgun-transport/blob/master/src/mailgun-transport.js), I dont see close method thats required in nodemailer.Transport (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nodemailer/index.d.ts#L210)